### ISSUE TITLE
[BUG]Invalid Plots. for no reason Plz help.

### ISSUE THREAD

**Issue by tangyisheng2** [Chia-Network/chia-blockchain#2140]
**Describe the bug**
Existing plots showing as invalid plots.
**To Reproduce**

1. Go to CHIA application on Windows.
2. Go to plots - add plot directionary
3. Existing plots show as invalid plots
Also try to add these plots via CLI, and it didn't work at all.

**Expected behavior**
The plots should start to farm.

**Screenshots**
![image](https://user-images.githubusercontent.com/16578638/115273561-491e5380-a172-11eb-8e42-08e6438b5580.png)
**Desktop (please complete the following information):**

- OS: Windows10
- OS Version/Flavor: [e.g. CentOS 7.2]
- CPU: Intel 6700k

**Additional context**

Logs from CLI:
00:46:51.155  chia.plotting.plot_tools         : ERROR    Failed to open file F:\CHIA_FIN\plot-k32-2021-04-16-16-51-f652fe833aabfd1bb67be71770c70bfa1405762c26c775d93b16548c17182ebd.plot. Invalid plot header magic Traceback (most recent call last):
  File "chia\plotting\plot_tools.py", line 185, in load_plots
ValueError: Invalid plot header magic

00:46:51.157  chia.plotting.plot_tools         : ERROR    Failed to open file F:\CHIA_FIN\plot-k32-2021-04-17-00-00-2739043ace7daeee952f9e4af7bd6e790f7dcc43d040d7113b2dd4f50256d5ac.plot. Invalid plot header magic Traceback (most recent call last):
  File "chia\plotting\plot_tools.py", line 185, in load_plots
ValueError: Invalid plot header magic

00:46:51.160  chia.plotting.plot_tools         : ERROR    Failed to open file F:\CHIA_FIN\plot-k32-2021-04-17-00-42-6a11533b8aa2cecc5fdd73f779045c22354fb83df7ebb76bb5b037231283a0ed.plot. Invalid plot header magic Traceback (most recent call last):
  File "chia\plotting\plot_tools.py", line 185, in load_plots
ValueError: Invalid plot header magic

00:46:51.163  chia.plotting.plot_tools         : ERROR    Failed to open file F:\CHIA_FIN\plot-k32-2021-04-17-06-39-8038dbef72c4cd09887c2363ea6efb19843d2a2c53547a33bd256b6c4406543e.plot. Invalid plot header magic Traceback (most recent call last):
  File "chia\plotting\plot_tools.py", line 185, in load_plots
ValueError: Invalid plot header magic

**Comment by wolzche** [comment]
my too, after forced to restart  the computer

**Comment by pulsar09** [comment]
same here, after restart of computer chia is not recognizing my plots
 ![image](https://user-images.githubusercontent.com/82962887/115586073-7e3eb900-a2cc-11eb-921a-c86a3fd29016.png)

**Comment by fiveangle** [comment]
I am seeing this too... `chia plots check` local on plotter where plots are created shows all good, dismount drive and sneakernet to harvester/farmer/fullnode and some of the plots show as "Invalid plot header magic".  This is on 1.1.2.  I have been using this method since 1.0.1 with expected results. Pool and Farmer keys match correctly.

**Comment by fiveangle** [comment]
update - it seems this may be related to plots that encounter an extended amount of time in the "cant write... waiting 5 min..." state during plotting.  once space is available the plot completes but then has this error later on a different system.  It is definitely a weird bug but I haven't nailed down exactly the trigger yet.

**Comment by hugoubeda** [comment]
I have the same problem, only with some plots, all created on the same machine, then moved to my harvester pc. I dont know why is this happening. My error is:

```
2021-04-26T22:05:17.572 harvester chia.plotting.plot_tools: ERROR    Failed to open file D:\CHIA FINAL\plot-k32-2021-04-25-17-21-4dd5976d4b11fa0ff46af9b294ef49bef4db80ac7c0c67f55016c6d8c1b2e0d2.plot. Invalid plot header magic Traceback (most recent call last):
  File "chia\plotting\plot_tools.py", line 183, in process_file
ValueError: Invalid plot header magic
```

I'm thinking it could be my origin path is on ext4 and my final path is on Windows, but this is only happening with some plots, not all, so the issue should be any other thing... Hope more info about this issue.

**Comment by hugoubeda** [comment]
> update - it seems this may be related to plots that encounter an extended amount of time in the "cant write... waiting 5 min..." state during plotting. once space is available the plot completes but then has this error later on a different system. It is definitely a weird bug but I haven't nailed down exactly the trigger yet.
> 
> also, to be clear, this was encountered on a xeon system running full ECC on ZFS file system so there is a greater chance of the sun going supernova tomorrow than there is of this being a "system error" ;)

Im thinking about your message. Are you copying your plots when they finish to another place while you still ploting? I plot on SSD Nvme for temp files and my final plot file goes to an HDD so, maybe if Im copying plots from my HDD to another external drive while I keep ploting, the HDD bandwidth will be full and the ploting process must wait, so later I have this error. I wait for your answer. Thanks!

**Comment by tangyisheng2** [comment]
> update - it seems this may be related to plots that encounter an extended amount of time in the "cant write... waiting 5 min..." state during plotting. once space is available the plot completes but then has this error later on a different system. It is definitely a weird bug but I haven't nailed down exactly the trigger yet.
> 
> also, to be clear, this was encountered on a xeon system running full ECC on ZFS file system so there is a greater chance of the sun going supernova tomorrow than there is of this being a "system error" ;)

Tried to plot using the cli, so far no errors anymore.

**Comment by fiveangle** [comment]
> > update - it seems this may be related to plots that encounter an extended amount of time in the "cant write... waiting 5 min..." state during plotting. once space is available the plot completes but then has this error later on a different system. It is definitely a weird bug but I haven't nailed down exactly the trigger yet.
> > also, to be clear, this was encountered on a xeon system running full ECC on ZFS file system so there is a greater chance of the sun going supernova tomorrow than there is of this being a "system error" ;)
> 
> Tried to plot using the cli, so far no errors anymore.

same binary creates the plots either way... the gui is unrelated (I am not using the gui)

**Comment by andrewreynolds2002** [comment]
I'm having the same problem with Maxio Shovel plotter. The first time I ran it, plots were coming out fast and they were 100% OK. Now I'm trying on different PC as well as the original PC where it worked just fine (Double checked keys etc) and now they are producing bad plots...Not sure how I went from making good plots to bad plots and keys were the same (copied from same config file/out of wallet)...mystery... help.
